### PR TITLE
perldelta 4fc2379: make a new stub to clone into when pushing a new pad

### PR DESCRIPTION
**Pull request by tonycoz**
If anyone has better wording for the second issue I'd love to see it.

### CODE REVIEW

File: pod/perldelta.pod [hunk 1]
```diff
@@ -476,7 +476,50 @@ manager will later use a regex to expand these into links.
...
+inner recursive call:
+
+    use v5.36.0;
+    sub outer ($x) {
+        my sub inner ($label) {
+            say "$label $x";
+        }
+        inner("first");
+        outer("inner") if $x eq "outer";
+        # $x visible inside this call to inside is incorrect
```
Comment (mauke) [inline pod/perldelta.pod]: Should be `call to inner`. But maybe rephrase as `this call to inner() sees the wrong $x`?

File: pod/perldelta.pod [hunk 2]
```diff
@@ -476,7 +476,50 @@ manager will later use a regex to expand these into links.
 
 =item *
 
-XXX
+Lexical subs now have a new stub in the pad for each recursive call
+into the containing function.  This fixed two problems:
```
Comment (mauke) [inline pod/perldelta.pod]: Not sure about tenses in English. Is it "This fixed" or "This fixes"?

**Comment by mauke** [comment]
The rest looks good to me.

File: pod/perldelta.pod (hunk 2 above)
Comment (khwilliamson) [reply pod/perldelta.pod]: s/fixed/fixes/

**Review by mauke — APPROVED** [review]
